cmd/skaffold/app/cmd/debug.go

```diff
@@ -29,7 +29,7 @@ import (
 // NewCmdDebug describes the CLI command to run a pipeline in debug mode.
 func NewCmdDebug() *cobra.Command {
 	return NewCmd("debug").
-		WithDescription("Run a pipeline in debug mode").
+		WithDescription("[beta] Run a pipeline in debug mode").
```

Comment:
Can we make the same changes here
https://github.com/GoogleContainerTools/skaffold/blob/master/docs/content/en/docs/references/cli/index_header#L20